### DISCUSSION TITLE
Welcome to filament-bookmarks-menu Discussions!

### DISCUSSION THREAD

**Discussion by andreaskviby** [STAFE-GROUP-AB/filament-bookmarks-menu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and features you all want.
  * Engage with other community members.

We want to have this as an open discussion about this plugin, maybe not bugs because they should be reported as issues here but more of a wishlist and showcase for the plugin itself. 

How do you use it? Tell us, please.

**Comment by ariesdg** [comment]
Sorry, newbie here, can you help me with this issue, I have installed this package to my linux server in AWS and once I bookmark a page, the menu_url apears to be local host instead of the server's IP/domain. Now if I access the app, I cannot access the bookmarked page cause it appears to be a broken page. 

<img width="351" alt="image" src="https://user-images.githubusercontent.com/51257627/226781534-90d2d6f9-7cb2-4d49-9455-7ea2fa408ccc.png">

**Comment by andreaskviby** [comment]
What is your app url in your env file?

**Reply by ariesdg** [reply]
Oh, my bad, it's actually localhost, thanks for pointing it out. It's now working. Thank you for prompt response.
